feeds/wlan-ap/opensync/files/bin/check_wan_link.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Tested this script on both my EX227 and EA8300 and it seemed to work as intended on both.